README.md

```diff
@@ -22,31 +20,38 @@ Thanks!
 If you're reading this at https://github.com/rails-api/active_model_serializers you are
 reading documentation for our `master`, which is not yet released.
 
-Please see below for the documentation relevant to you.
-
-- [0.10 (0-10-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-10-stable)
-- [0.10.6 (latest release) Documentation](https://github.com/rails-api/active_model_serializers/tree/v0.10.6)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/gems/active_model_serializers/0.10.6)
-  - [Guides](https://github.com/rails-api/active_model_serializers/tree/v0.10.6/docs)
-- [0.9 (0-9-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-9-stable)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-9-stable)
-- [0.8 (0-8-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-8-stable)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/rails-api/active_model_serializers/0-8-stable)
+<table>
+  <tr>  
+    <td> [0.10 (0-10-stable) Documentation](https://github.com/rails-api/active_model_serializers/tree/0-10-stable) </td>
```

Comment:
this looks better in atom. :-\

Reply:
![2017-09-19-104959_486x186_scrot](https://user-images.githubusercontent.com/199018/30598645-513f0e08-9d28-11e7-966f-23470d23ced1.png)